### ISSUE TITLE
bug: search parameters are trimmed from the URL

### ISSUE THREAD

**Issue by vjo** [ionic-team/stencil-playwright#45]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil-playwright/blob/main/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil-playwright/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil-playwright/issues) that already report this problem, without success.

### Stencil Playwright Version

v0.2.0

### Current Behavior

Navigating to a URL containing search parameters (e.g. `https://playwright.dev/?key=val`) with `goto` will remove the search parameters.

### Expected Behavior

Navigating to a URL containing search parameters (e.g. `https://playwright.dev/?key=val`) with `goto` should keep the parameters in the URL.

### Steps to Reproduce

```js
import { expect } from '@playwright/test';
import { test } from '@stencil/playwright';

test('Playwright', async ({ page }) => {
  const url = 'https://playwright.dev/?key=val';
  await page.goto(url);

  expect(page.url()).toBe(url); // ❌
});
```

### Code Reproduction URL

None

**Comment by tanner-reits** [comment]
Addressed by #46 and will be included in the next release! Thanks @vjo 😄

**Comment by tanner-reits** [comment]
A fix for this was included in today's [v0.2.1](https://github.com/ionic-team/stencil-playwright/releases/tag/v0.2.1) release!
